### ISSUE TITLE
Φαρδιά η αριστερή στήλη

### ISSUE THREAD

**Issue by dimitrisdervas** [dimitrisdervas/lilavillas#8]
Καλύτερα;

**Comment by rovithakis** [comment]
Ναι, το πλάτος είναι καλό. Ίσως μεγαλώσουμε λίγο τα γράμματα, αλλά άσε να το δει και ο Νώντας - λείπει στο εξωτερικό...

**Comment by rovithakis** [comment]
Νομίζω είναι εντάξει, αν και θα προτιμούσα  πιο μικρά τα: contact, reservation, view gallery
N.
